### DISCUSSION TITLE
number of markers per cluster are less

### DISCUSSION THREAD

**Discussion by GreyRockIQ** [satijalab/seurat#5928] [Q&A]
Hello
I am observing the number of markers identified per a given cluster are very less(6 or 28) though the no. of cells in cluster have high (3000 cells) in number(with nFeatures>2000) when 2000 features are taken for variable feature selection in integrated analysis.

Is there anything I can change/set to observe more features per cluster with FindMarkers function in integrated analysis?

Thanks
GreyRock

**Comment by timoast** [comment]
You can try lowering the `logfc.threshold` and `min.pct` in `FindMarkers()`
